### PR TITLE
Optimization cleanDir

### PR DESCRIPTION
**Pull request by ping035627**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
Optimization cleanDir.

Which issue(s) this PR fixes:

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

/release-note-none
/priority backlog

### CODE REVIEW

File: cmd/kubeadm/app/cmd/reset.go [hunk 1]
```diff
@@ -266,26 +267,17 @@ func removeContainers(execer utilsexec.Interface, criSocketPath string) error {
...
-		return nil
-	}
-
-	d, err := os.Open(filePath)
-	if err != nil {
-		return err
-	}
-	defer d.Close()
-	names, err := d.Readdirnames(-1)
+	names, err := ioutil.ReadDir(filePath)
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/reset.go]: if the directory does not exist ReadDir will throw an error, see:
https://play.golang.org/p/x6JlQivxSVY

cleanDir should return without error in such cases.

**Review by neolit123 — COMMENTED** [review]
hi, @ping035627 
i've added one comment related to the change.

/hold

File: cmd/kubeadm/app/cmd/reset.go (hunk 1 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/reset.go]: i don't think we have to modify cleanDir as it does not need optimization.

Comment (yagonobre) [reply cmd/kubeadm/app/cmd/reset.go]: +1

**Comment by ping035627** [comment]
I see, I will close this pr, thanks for reminding.

**Pull request closed without merging**